lib/resque/worker.rb

```diff
@@ -347,22 +347,10 @@ def reserve
       raise e
     end
 
-    # Reconnect to Redis to avoid sharing a connection with the parent,
+    # Disconnect from Redis to avoid sharing a connection with the parent,
     # retry up to 3 times with increasing delay before giving up.
-    def reconnect
```

Comment:
Same here with this one since it's still technically a public method. If we can make an alias or an alternative method that would be ideal.

Also in this case, does it make sense to attempt the `PING` after disconnecting in order to retain the reconnect/retry logic? This is the only place resque seems to do this and I'm wondering if removing it might result in reduced fault tolerance. If it only disconnects and then we rely on the redis client to handle reconnecting, you may find connection errors at interesting new spots in your backtrace that you weren't expecting.

Reply:
> retain the reconnect/retry logic?

That reconnect/retry logic is totally redundant since Redis has it baked in, but ok. I honestly don't care, I don't use Resque.

Reply:
Yeah I hear ya... it was more out of concern for changing the behavior on public APIs since I know of other people using this that have various hooks and plugins that rely on this stuff. I appreciate you tweaking your PR though and I'm going to get a build going for early next week after Labor Day. Thanks!!